esg_init.py

```diff
@@ -76,8 +76,8 @@ def init():
     openssl_version = "0.9.8r"
     openssl_min_version = "0.9.8e"
     openssl_max_version = "0.9.9z"
-    java_version = "1.8.0_162"
-    java_min_version = "1.8.0_162"
+    java_version = "1.8.0_181"
+    java_min_version = "1.8.0_181"
```

Comment:
I am looking to merge this in, but @pchengi2 seems to have gone to latest release ( _192) in  2.8, based on an email he sent.
```suggestion
    java_version = "1.8.0_192"
    java_min_version = "1.8.0_192"
```